server/channels/api4/system.go

```diff
@@ -194,7 +194,7 @@ func getSystemPing(c *Context, w http.ResponseWriter, r *http.Request) {
 
```

Comment:
I can't comment above, but I think the issue is in L182 where we don't actually log the `appErr`. So there is no indication of exactly what went wrong.

Reply:
Right.. I totally forget that part :)